app/views/providers/legal_aid_applications/_application_list.html.erb

```diff
@@ -4,13 +4,13 @@
 
         table.with_head do |head|
           head.with_row do |row|
-            row.with_cell do
+            row.with_cell(html_attributes: { "aria-sort": "none" }) do
               tag.span(class: "aria-sort-description") do
                 concat t(".applicant_name")
                 concat tag.span(class: "sort-direction-indicator")
               end
             end
-            row.with_cell do
+            row.with_cell(html_attributes: { "aria-sort": "descending" }) do
```

Comment:
Non-blocking question: do we change this in the JS when a user clicks?

Reply:
This is now handled by the moj design system sortable table. When you click a heading (to sort the table), `aria-sort` is updated